### ISSUE TITLE
R171 introduces a regression

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gyc2432968/python-twitter#77]
```
The new URLs in R171 for GetUserTimeline should not be xml urls, but json.
```

Original issue reported on code.google.com by `a.lind...@gmail.com` on 19 Jun 2009 at 9:42

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by r172.
```

Original comment by `dclinton` on 20 Jun 2009 at 4:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in R172.  Please verify.  Thanks for catching this!
```

Original comment by `dclinton` on 20 Jun 2009 at 4:12

**Comment by GoogleCodeExporter** [comment]
```
No problem. Tested, and it works--it was the same change I made locally.

Found it while working on some test cases for http://github.com/atl/twitstream ,
which may be of some interest.
```

Original comment by `a.lind...@gmail.com` on 20 Jun 2009 at 4:27

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0f3a9eb7a8.
```

Original comment by `dclinton` on 24 Oct 2009 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 2eff39b31e.
```

Original comment by `dclinton` on 24 Oct 2009 at 4:58
